### PR TITLE
Add AWS VPC Flow Log integration

### LINKED ISSUE #642: [INTEGRATION] AWS VPC Flow Log Integration
**Is your feature request related to a problem?**
The integration project doesn't have content to demonstrate how AWS VPC flow log looks like in Opensearch Dashboard

**What solution would you like?**
Create VPC Flow log schema, list of visual and query based assets including the dashboard, sample data.

**What alternatives have you considered?**
N/A

**Do you have any additional context?**
https://docs.aws.amazon.com/solutions/latest/centralized-logging-with-opensearch/vpc-flow-logs.html#sample-dashboard-6

### PR DESCRIPTION
**Pull request by wanghd89**
### Description
Create VPC Flow log schema, list of visual and query based assets including the dashboard, sample data.

### Issues Resolved
#642 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        "namespace": "production",
+        "type": "logs"
+      }
+    },
+    "cloud": {
+      "provider": "aws",
+      "account": {
+        "id": "443273586136"
+      },
+      "platform": "aws_vpc",
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json:22]: Trailing comma

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    "aws": {
+      "s3": {
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json:25]: `aws.s3` is not defined in any of the provided mapping files, so this data would just be dropped if a user tried to load this sample.

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        "key": "AWSLogs/530679726453/vpcflowlogs/us-east-2/2023/01/28/530679726453_vpcflowlogs_us-east-2_fl-023c6afa025ee5a04_20230128T0930Z_3a9dfd9d.log.gz"
+      },
+      "vpc": {
+        "version" : "2",
+        "account-id" : "530679726453",
+        "interface-id" : "eni-0e250409d410e1290",
+        "srcaddr" : "162.142.125.177",
+        "dstaddr" : "10.0.0.200",
+        "srcport" : "38471",
+        "dstport" : "12313",
```
[the remark this answers is not in the document]
Comment (wanghd89) [reply server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json:36]: fixed

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        "action" : "ACCEPT",
+        "log-status" : "OK"
+      }
+    },
+    "communication": {
+      "source": {
+        "address": "162.142.125.177",
+        "port": "38471",
+        "packets": "1",
+        "bytes" : "44"
```
[the remark this answers is not in the document]
Comment (wanghd89) [reply server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json:51]: fixed

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json (hunk 1 above)
Comment (wanghd89) [reply server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json:22]: removed

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json (hunk 2 above)
Comment (wanghd89) [reply server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json:25]: ok, updated the s3-1.0.0 mapping for aws.s3

**Review by Swiddis — APPROVED** [review]

**Review by YANG-DB — APPROVED** [review]

**Pull request merged**